src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1074,7 +1073,7 @@ bool entitiesQuery
     connection = getMongoConnection();
     try
     {
-        if ((details == true) && (countP != NULL))
+        if (countP != NULL)
```

Comment:
It is only done if countP != NULL.
countP is NULL unless '?count=true' is set.
It works just like before.

NTC